lib/gui/models/store.js

```diff
@@ -90,9 +89,17 @@ const storeReducer = (state, action) => {
 
       const newState = state.set('availableDrives', Immutable.fromJS(action.data));
 
-      if (action.data.length === 1) {
+      // Non-system drives
+      const noSystemDrives = _.filter(action.data, (drive) => {
```

Comment:
In preparation for the addition of "Boot drive" detection (amongst other filtering), could this `noSystemDrives` be renamed to `removableDrives` ?